app/models/concerns/hostext/token.rb

```diff
@@ -35,6 +35,11 @@ def token_expired?
       self.token.expires < Time.now.utc
     end
 
+    def valid_token?
```

Comment:
Where is this used?

Reply:
That's a leftover from the old version. Removed.